### ISSUE TITLE
httpd pod fails readiness probe

### ISSUE THREAD

**Issue by barnzdan** [ManageIQ/manageiq-pods#234]
```oc v3.6.0-alpha.0+0343989
kubernetes v1.5.2+43a9be4
features: Basic-Auth
Server https://openshift:8443
openshift v1.4.1+3f9807a
kubernetes v1.4.0+776c994```

When standing up the miq stack the httpd pod continues to fail the readiness probe. When using a remote shell I can see there is no a broken link that httpd expects, /etc/httpd/logs -> /var/log/httpd, where /var/log/httpd is missing. Further, I dont have the permission to perform systemctl within the pod. I am working off the master branch of this repo.

Below are details of the scc commands I've run per README for Openshift and the pod description:

oc adm policy add-scc-to-user anyuid system:serviceaccount:manageiq:miq-anyuid
oc adm policy add-scc-to-user anyuid system:serviceaccount:manageiq:miq-orchestrator
oc adm policy add-scc-to-user privileged system:serviceaccount:manageiq:miq-privileged
oc adm policy add-scc-to-user anyuid system:serviceaccount:manageiq:miq-httpd
oc policy add-role-to-user view system:serviceaccount:manageiq:miq-orchestrator -n manageiq
oc policy add-role-to-user edit system:serviceaccount:manageiq:miq-orchestrator -n manageiq

httpd pod:
% oc describe po/httpd-1-nk5e9
Name:			httpd-1-nk5e9
Namespace:		manageiq
Security Policy:	anyuid
Node:			node05/10.3.96.158
Start Time:		Wed, 18 Oct 2017 11:12:02 -0400
Labels:			app=manageiq
			deployment=httpd-1
			deploymentconfig=httpd
			name=httpd
Status:			Running
IP:			10.129.3.71
Controllers:		ReplicationController/httpd-1
Containers:
  httpd:
    Container ID:	docker://5d0d47163169b82bdef44e3cf9f59c23a4849cc097f802787874c87450e2d608
    Image:		docker.io/manageiq/httpd:latest
    Image ID:		docker-pullable://docker.io/manageiq/httpd@sha256:a32092a5c48051ff3a89188fdf23558d8f60fcf8ea411c551031c7cadcc202ba
    Ports:		80/TCP, 8080/TCP
    Limits:
      memory:	8Gi
    Requests:
      cpu:		500m
      memory:		512Mi
    State:		Running
      Started:		Wed, 18 Oct 2017 11:14:04 -0400
    Last State:		Terminated
      Reason:		Error
      Exit Code:	137
      Started:		Wed, 18 Oct 2017 11:13:14 -0400
      Finished:		Wed, 18 Oct 2017 11:14:02 -0400
    Ready:		False
    Restart Count:	2
    Liveness:		exec [pidof httpd] delay=15s timeout=3s period=10s #success=1 #failure=3
    Readiness:		tcp-socket :80 delay=10s timeout=3s period=10s #success=1 #failure=3
    Volume Mounts:
      /etc/httpd/auth-conf.d from httpd-auth-config (rw)
      /etc/httpd/conf.d from httpd-config (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from miq-httpd-token-3f9zj (ro)
    Environment Variables:
      HTTPD_AUTH_TYPE:			<set to the key 'auth-type' of config map 'httpd-auth-configs'>
      HTTPD_AUTH_KERBEROS_REALMS:	<set to the key 'auth-kerberos-realms' of config map 'httpd-auth-configs'>
Conditions:
  Type		Status
  Initialized 	True 
  Ready 	False 
  PodScheduled 	True 
Volumes:
  httpd-config:
    Type:	ConfigMap (a volume populated by a ConfigMap)
    Name:	httpd-configs
  httpd-auth-config:
    Type:	ConfigMap (a volume populated by a ConfigMap)
    Name:	httpd-auth-configs
  miq-httpd-token-3f9zj:
    Type:	Secret (a volume populated by a Secret)
    SecretName:	miq-httpd-token-3f9zj
QoS Class:	Burstable
Tolerations:	<none>
Events:
  FirstSeen	LastSeen	Count	From						SubObjectPath		Type		Reason		Message
  ---------	--------	-----	----						-------------		--------	------		-------
  2m		2m		1	{default-scheduler }					Normal		Scheduled	Successfully assigned httpd-1-nk5e9 to node05
  2m		2m		1	{kubelet node05}	spec.containers{httpd}	Normal		Created		Created container with docker id a5a6d5d56f9a; Security:[seccomp=unconfined]
  2m		2m		1	{kubelet node05}	spec.containers{httpd}	Normal		Started		Started container with docker id a5a6d5d56f9a
  55s		55s		1	{kubelet node05}	spec.containers{httpd}	Normal		Killing		Killing container with docker id a5a6d5d56f9a: pod "httpd-1-nk5e9_manageiq(b1a60fbd-b416-11e7-a733-005056876455)" container "httpd" is unhealthy, it will be killed and re-created.
  53s		53s		1	{kubelet node05}	spec.containers{httpd}	Normal		Started		Started container with docker id c06e2eefe523
  53s		53s		1	{kubelet node05}	spec.containers{httpd}	Normal		Created		Created container with docker id c06e2eefe523; Security:[seccomp=unconfined]
  1m		35s		4	{kubelet node05}	spec.containers{httpd}	Warning		Unhealthy	Liveness probe failed: 
  1m		5s		10	{kubelet node05}	spec.containers{httpd}	Warning		Unhealthy	Readiness probe failed: dial tcp 10.129.3.71:80: getsockopt: connection refused
  2m		5s		3	{kubelet node05}	spec.containers{httpd}	Normal		Pulling		pulling image "docker.io/manageiq/httpd:latest"
  5s		5s		1	{kubelet node05}	spec.containers{httpd}	Normal		Killing		Killing container with docker id c06e2eefe523: pod "httpd-1-nk5e9_manageiq(b1a60fbd-b416-11e7-a733-005056876455)" container "httpd" is unhealthy, it will be killed and re-created.
  2m		4s		3	{kubelet node05}	spec.containers{httpd}	Normal		Pulled		Successfully pulled image "docker.io/manageiq/httpd:latest"
  3s		3s		1	{kubelet node05}	spec.containers{httpd}	Normal		Created		Created container with docker id 5d0d47163169; Security:[seccomp=unconfined]
  3s		3s		1	{kubelet node05}	spec.containers{httpd}	Normal		Started		Started container with docker id 5d0d47163169

**Comment by carbonin** [comment]
Just at a first glance, we require Origin 1.5 or higher (ref: https://github.com/ManageIQ/manageiq-pods#prerequisites)

Not sure if that could be causing this. @fbladilo ?

**Comment by fbladilo** [comment]
@barnzdan @carbonin We do require Origin 1.5 at minimum, since some objects referenced by template are only available in 1.5 (I.E StatefulSets).

**Comment by barnzdan** [comment]
The problem with the httpd probe not working was resolved by doing the following:

1. Remove the httpd pod docker images from all cluster nodes
2. yum update oci-systemd-hook (on all nodes)

Would still need to upgrade the cluster to 1.5 in order to use StatefulSets.
